vector/src/debug/java/im/vector/app/features/debug/settings/DebugPrivateSettingsViewModel.kt

```diff
@@ -44,11 +44,12 @@ class DebugPrivateSettingsViewModel @AssistedInject constructor(
     }
 
     private fun observeVectorDataStore() {
-        vectorDataStore.forceDialPadDisplayFlow.setOnEach {
-            copy(dialPadVisible = it)
+        debugVectorOverrides.forceDialPad().setOnEach {
+            copy(
+                    dialPadVisible = it
+            )
```

Comment:
Not something to action as this is more of a personal preference thing but I never understood opting for this format rather than a single line `copy(dialPadVisible = it)`. It just personally looks cleaner to me

Reply:
Yes, when there is only one line it's OK. I think, it's for future hypothetical code to split into 3 lines, so if for instance in the future we have:
```kotlin
copy(
    dialPadVisible = it,
    other = blah
)
```

But this is definitely not a strong constraint.

Reply:
I don't have any strong feelings for either, I chose to stick with the consistency of the file, happy to change!